poetry/core/masonry/utils/include.py

```diff
@@ -27,6 +27,10 @@ def __init__(
 
         self._elements = sorted(list(self._base.glob(str(self._include))))
 
+    @property
+    def include(self):  # type: () -> Path
+        return self._include
+
```

Comment:
Unless I am missing something, this is not strictly needed for this change, right? It's possible to just use `self._include` below.

Reply:
Indeed. I added the @property because it seemed to be the pattern in other parts of the code.